README.md

```diff
@@ -42,6 +42,14 @@ Current requirements are:
   ```
 This will fill the database with the latest data from ergast, and create the initial test database so all future tests will run quicker.
 
+## Running the Server
+
+To run the server
+
+```python manage.py runserver 0.0.0.0:8000```
+
+This will start Jolpica on port 8000. Once it is running you can access the API via browser, curl or other API testing tool like Postman or Bruno with `http://0.0.0.0:8000/ergast/f1/`
```

Comment:
```suggestion
This will start Jolpica on port 8000. Once it is running you can access the API via browser, curl or other API testing tool like Postman or Bruno with `http://localhost:8000/ergast/f1/`
```